src/item.cpp

```diff
@@ -238,7 +238,12 @@ void Item::setID(uint16_t newid)
 	id = newid;
 
 	const ItemType& it = Item::items[newid];
-	uint32_t newDuration = it.decayTime * 1000;
+	uint32_t newDuration;
+	if (it.decayTimeMax != 0) {
+		newDuration = normal_random(it.decayTime, it.decayTimeMax) * 1000;
+	} else {
+		newDuration = it.decayTime * 1000;
+	}
```

Comment:
You don't need this branch, if min and max are the same, `normal_random` will always return that value

Reply:
> You don't need this branch, if min and max are the same, `normal_random` will always return that value

Do you mean IF this way?
```cpp
if (it.decayTimeMax != it.decayTime) { ... }
```

Reply:
No if.

```
uint32_t newDuration = normal_random(it.decayTime, it.decayTimeMax) * 1000;
```

In the case that `it.decayTime == it.decayTimeMax` it will just return that value.

Speaking of which, it's probably wise to rename `decayTime` to `decayTimeMin`.